src/kubectl-helm-minikube/install.sh

```diff
@@ -122,6 +122,13 @@ case $architecture in
     *) echo "(!) Architecture $architecture unsupported"; exit 1 ;;
```

Comment:
Can you bump the patch version of the Feature? 

ℹ️ Once the PR is merged, the action automatically publishes new version of this Feature (**provided the version is updated). We follow semver rules for bumping versions.